python/src/trezorlib/ui.py

```diff
@@ -95,8 +95,10 @@ def get_pin(self, code=None):
                 pin = prompt("Please enter {}".format(desc), hide_input=True)
             except click.Abort:
                 raise Cancelled from None
-            if not pin.isdigit():
-                echo("Non-numerical value provided, please try again")
+            if any(d not in "123456789" for d in pin):
```

Comment:
please change this check also in `client.py` `_callback_pin`

Reply:
Done https://github.com/trezor/trezor-firmware/pull/906/commits/4d46fd078e50f01ec53c7bed2ced7caa472dc4b7